### DISCUSSION TITLE
Saving Data

### DISCUSSION THREAD

**Discussion by KamlaXPL** [BG-Software-LLC/SuperiorSkyblock2#1263] [Implemented Feature Requests] [answered]
Add saving mission data and other add-ons every so often in particular oneblock, because all it takes is a server crash or whatever and players have things set back a few days, or at least a command for that.

It would be nice if you added an api to oneblock, because currently you cant download any data from within your own plugin, because when they load a module from your plugin, bukkit doesnt detect the class.

**Comment by OmerBenGera** [comment, marked as answer]
Missions progress is saved every 5 minutes. About OneBlock, saved every interval as well.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
